modules/client-configuration/pages/clients-sleses.adoc

```diff
@@ -0,0 +1,170 @@
+
+Use the [command]``cat`` or [command]``tail -f`` command to view the [path]``/var/log/rhn/reposync/channel-label.log`` file.
+
+If you use this method, remember that base channels can contain multiple child channels.
+
+Each of the child channels will generate its own log during the synchronization progress.
+
+You will need to check all the base and child channel log files to be sure that the synchronization is complete.
+
+You will also need an activation key associated with the correct {slesesa} channels (6 or 7)
+
+For more information on activation keys, see xref:client-configuration:clients-and-activation-keys.adoc[].
+
+=== Add {rhel} or {centos} Media
+
+The Base {slesesa} channel does not contain any packages.
+
+This is because {suse} does not provide {rhel} or {centos} base media.
+
+You will need to obtain base media from {redhat} or {centos}, which you can then add as a child channel to the {slesesa} parent channel.
+
+The following procedure guides you through setup of the {rhel} or {centos} media as a {productname} channel.
+
+All packages on the media will be mirrored into a child channel located under the distribution name and architecture.
+
+.Procedure: Setting up a {rhel} or {centos} Channel
+
+. In the {productname} {webui}, navigate to menu:Channels[Manage Software Channels], and click btn:[Create Channel].
+. Fill in the channel details, and add the channel as a child to the corresponding {rhel} or {centos} distribution channel for your architecture.
+The base parent channel will not contain any packages.
+. Modify the associated activation key to include your new child channel.
+. At the command prompt, as root, copy your installation disk image to the [path]``/tmp`` directory.
+. Create a directory to contain the media content (you can replace [path]``/rhel`` with [path]``/centos`` if you are using {centos}
++
+----
+mkdir -p /srv/www/htdocs/pub/rhel
+----
+. Mount the ISO:
++
+----
+mount -o loop /tmp/name_of_iso /srv/www/htdocs/pub/rhel
+----
+. Synchronize the packages with [command]``spacewalk-repo-sync``:
++
+For {rhel} or {centos} 7:
++
+----
+spacewalk-repo-sync -c channel_name -u https://127.0.0.1/pub/rhel/
+Repo URL: https://127.0.0.1/pub/rhel/
+Packages in repo:              [...]
+Packages already synced:       [...]
+Packages to sync:              [...]
+[...]
+----
++
+For {rhel} or {centos} 6:
++
+----
+spacewalk-repo-sync -c channel_name -u https://127.0.0.1/pub/rhel/Server/
+Repo URL: https://127.0.0.1/pub/rhel/Server/
+Packages in repo:              [...]
+Packages already synced:       [...]
+Packages to sync:              [...]
+[...]
+----
+
+Sometimes, the [command]``spacewalk-repo-sync`` will stop running during a synchronization, which will give this error:
+----
+[Errno 256] No more mirrors to try.
+----
+
+If this occurs, you can run [command]``spacewalk-repo-sync`` in debugging mode to determine the error.
+
+Start debugging mode:
```

Comment:
There is doubt whether this will still work: see mc's comment https://github.com/SUSE/doc-susemanager/pull/757